dll/win32/xolehlp/CMakeLists.txt

```diff
@@ -1,5 +1,4 @@
 
-add_definitions(-D__WINESRC__)
 include_directories(BEFORE ${REACTOS_SOURCE_DIR}/sdk/include/reactos/wine)
```

Comment:
Shouldn't this `include_directories()` be removed too? (Everywhere.)

Reply:
OK. Thanks.